client/extensions/wp-job-manager/components/navigation/index.jsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { find, get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import SectionNav from 'components/section-nav';
+import SectionNavTabs from 'components/section-nav/tabs';
+import SectionNavTabItem from 'components/section-nav/item';
+import sectionsModule from 'sections';
+import { Tabs } from '../../constants';
+
+class Navigation extends Component {
+	static propTypes = {
+		activeTab: PropTypes.string,
+		site: PropTypes.object,
+	};
+
+	static defaultProps = {
+		activeTab: '',
+	};
+
+	getLabel( slug ) {
+		const { JOB_LISTINGS, JOB_SUBMISSION, PAGES } = Tabs;
+
+		switch ( slug ) {
+			case JOB_LISTINGS.slug:
+				return JOB_LISTINGS.label;
+			case JOB_SUBMISSION.slug:
+				return JOB_SUBMISSION.label;
+			case PAGES.slug:
+				return PAGES.label;
+		}
+	}
+
+	getTabs() {
+		const tabs = [];
+
+		for ( const key in Tabs ) {
+			if ( Tabs.hasOwnProperty( key ) ) {
+				tabs.push( Tabs[ key ] );
+			}
+		}
+
+		return tabs;
+	}
+
+	getSettingsPath() {
+		const sections = sectionsModule.get();
+		const section = find( sections, ( value => value.name === 'wp-job-manager' ) );
+
+		return get( section, 'settings_path' );
+	}
+
+	renderTabItems( tabs ) {
+		const { slug: listingsSlug } = Tabs.JOB_LISTINGS;
+		const { activeTab, site } = this.props;
+
+		return tabs.map( ( { slug: tabSlug } ) => {
```

Comment:
Couldn't we cut down on some lookup helpers (such as `getLabel`) if we used lodash's `map` here to iterate directly over the `Tabs` object?